smartdevicelink/sdl_java_suite
# Fix issues in sendRPCs and sendSequentialRPCs

Fixes #1061

This PR is **ready** for review.

### Risk
This PR makes **minor** API changes.

### Summary
This PR fixes some issue in `sendRPCs` and `sendSequentialRPCs` methods:
- Calls the `onResponse/onError` for every RPC's `OnRPCResponseListener` 
- Calls the `onResponse/onError` in the `OnMultipleRequestListener` for every RPC 
- Continue sending RPCs if one failed
- `onUpdate` in `OnMultipleRequestListener`  is called for every RPC incuding the failed ones 

### Testing Plan
- Create a list of RPCs that you want to send and make sure one of them at least fails. The one that fails should not be the last one.
- Set an `OnRPCResponseListener` listener for all RPCs
- Send the RPCs using `sendSequentialRPCs`
- Make sure that `onResponse/onError` is called for every RPC in both the RPC's `OnRPCResponseListener` and the `OnMultipleRequestListener`
- Make sure that `onUpdate` is called for every RPC in the `OnMultipleRequestListener` and that `onFinished()` is called once in the end
- Repeat the same test using `sendRPCs`
- Repeat the tests again for both `sendRPCs` and `sendSequentialRPCs` in JavaSE library

### CLA
